execution_engine/src/core/engine_state/genesis.rs

```diff
@@ -235,6 +241,9 @@ impl ExecConfig {
         accounts: Vec<GenesisAccount>,
         wasm_config: WasmConfig,
         validator_slots: u32,
+        auction_delay: u64,
+        initial_era_id: EraId,
```

Comment:
Can't we make that always 0? (If not, we probably need to pass it into the era supervisor as well.)

Reply:
If we want to have `initial_era_id` as always zero, in my opinion we should retain it as a constant and then not have it in the chainspec. 
@henrytill

Reply:
Absolutely fine with me if we leave it as a constant! That said, I wasn't the one who suggested it be moved into the chainspec.

cc @EdHastingsCasperLabs

Reply:
Perfect, that one was the only I was the most iffy about, I will yank it out

Reply:
If everyone is okay with the first era being era 0, let's keep it a constant.

Otherwise please make sure we have tickets to make the consensus component work with other settings.

Reply:
I have reverted to it back to being a constant value at 0